editor/static/sharing/index.js

```diff
@@ -0,0 +1,12 @@
+function share(program){
+    if (program =="whatsapp"){
+        var whatsapp = "https://web.whatsapp.com/send?text=```";
+        var text = savedTabInfo[focusedTab].text;
+        console.log(text);
+        window.open(whatsapp + text + "```" + "%0a%0a This was shared via Notepad--");
+    }
+    else if (program=="mail")
+        window.open('mailto:test@example.com?subject=This was shared via Notepad--&body=' +savedTabInfo[focusedTab].text);
+    else if (program=="reddit")
+        window.open('https://www.reddit.com/r/ProgrammerHumor/submit?selftext=true')
```

Comment:
Does there need to be text here or not? It would be nice if it opens the reddit page and prefills the text with the textarea text but not sure if this is possible.

Reply:
Same holds for tweets